### DISCUSSION TITLE
Ergast changes

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#388] [Q&A] [answered]
Apologies for being so heavy, but before this worked perfectly with fastf1 v:2.3.3,
```
Url="http://ergast.com/api/f1/"+str(AÑO)+"/"+str(GPNº)+"/results"

response = requests.get(Url)
if response.status_code == 200: # 200 = cód. HTTP de ejecución correcta
    data = response.json()['MRData']['RaceTable'] # Extraer los datos
    Correcto1 = True
else:
    print('Error: Fallo en la recepción de [RaceTable] desde Ergast API.')
    Correcto1 = False
    
if Correct1 :   
    RESULTADOS = data['Races'][0]['Results']
    AÑO = data['season']
    GPNº = data['round']
    Evento = data['Races'][0]['raceName']
    
    print("\n", AÑO, GPNº, Evento, " RESULTADOS DE CARRERA : \n")
    print(RESULTADOS[0]) # print primer registro
```
but with fastf1 v:3.0.3 it gives:
```
data = response.json()['MRData']['RaceTable'] # Extraer los datos

   File ~\anaconda3\envs\FF1_V3\lib\site-packages\requests\models.py:975 in json
     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)

JSONDecodeError: Expecting value
```

I'm messing with the joint changes in fasft1 and ergast, how to fix it?

In advance, thank you very much.

**Comment by theOehrly** [comment, marked as answer]
Hi, first off, none of the code that you have posted is making use of FastF1. So this error is completely unrelated to FastF1. 
Second, Ergast is currently having server problems due to larger scale server maintenance. This is causing the problems you are seeing (also discussed in #384 in the context of FastF1).

Additionally, there is one specific error where Ergast returns status=200, "Unable to select database". Therefore, the response is not treated as an error in your code above, but the content of the response is not valid JSON.

**Reply by McHouse** [reply]
Oooopppsss... sorry for the confusion, no doubt caused by the fact that I use both tools together, obviously for the purposes of Formula 1 fans.
I will also review #384, to see if I am able to draw any conclusions.
Sorry, I'll try to be more careful going forward. Apologies.
And once again, Thank you very much 'theOehrly', thank you for the speed, for this your work, and for your time.
I promise to buy you a coffee, but not at Harrods. :D

**Reply by theOehrly** [reply]
No worries, it's easy to get lost sometimes. And you're not the only one confused by the current problems with Ergast😅
